cypress/fixtures/objects/validationObjects/accessibility.json

```diff
@@ -269,61 +283,129 @@
-                "type": "fixture",
+                "type": "undefined",
                 "validations": [
                     {
-                        "mode": "staticContentValidation",
-                        "type": "ACCESSIBILITY_FONTCOLOR_FFFFFF",
-                        "description": "Validation of the accessibility fontColor ffffff"
+                        "field": "result.styles.fontColor",
+                        "description": "Validation of the accessibility fontColor undefined"
                     }
                 ]
             }
         ]
     },
-    "ACCESSIBILITY_FONTEDGECOLOR_7F7F7F": {
+    "ACCESSIBILITY_FONTEDGECOLOR_UNDEFINED": {
         "method": "accessibility.closedCaptionsSettings",
         "data": [
             {
-                "type": "fixture",
+                "type": "undefined",
                 "validations": [
                     {
-                        "mode": "staticContentValidation",
-                        "type": "ACCESSIBILITY_FONTEDGECOLOR_7F7F7F",
-                        "description": "Validation of the accessibility fontEdgeColor 7f7f7f"
+                        "field": "result.styles.fontEdgeColor",
+                        "mode": "",
+                        "type": "",
+                        "description": "Validation of the accessibility fontEdgeColor undefined"
                     }
                 ]
             }
         ]
     },
-    "ACCESSIBILITY_BACKGROUNDCOLOR_000000": {
+    "ACCESSIBILITY_FONTOPACITY_UNDEFINED": {
         "method": "accessibility.closedCaptionsSettings",
         "data": [
             {
-                "type": "fixture",
+                "type": "undefined",
                 "validations": [
                     {
-                        "mode": "staticContentValidation",
-                        "type": "ACCESSIBILITY_BACKGROUNDCOLOR_000000",
-                        "description": "Validation of the accessibility backgroundColor 000000"
+                        "field": "result.styles.fontOpacity",
+                        "description": "Validation of the accessibility fontEdgeOpacity undefined"
                     }
                 ]
             }
         ]
     },
-    "ACCESSIBILITY_WINDOWCOLOR_000000": {
+    "ACCESSIBILITY_BACKGROUNDCOLOR_UNDEFINED": {
         "method": "accessibility.closedCaptionsSettings",
         "data": [
             {
-                "type": "fixture",
+                "type": "undefined",
                 "validations": [
                     {
-                        "mode": "staticContentValidation",
-                        "type": "ACCESSIBILITY_WINDOWCOLOR_000000",
-                        "description": "Validation of the accessibility windowColor 000000"
+                        "field": "result.styles.backgroundColor",
+                        "description": "Validation of the accessibility backgroundColor undefined"
+                    }
+                ]
+            }
+        ]
```

Comment:
why do we need to create new objects from 344- 400 ? Those test cases would have some set of validation objects already. Cant we use that?

Reply:
That's because whichever new validation objects of type undefined that are added are the cases which do not actually have a corresponding validation object in the respective module file. For example, if we take the case of backgroundOpacity, in current dev, it doesn't have a validation object of a certain type in cypress/fixtures/objects/validationObjects/accessibility.json rather its default value for validation is INTEGER12 present in defaultTestData.json, thats why as part of the ticket a new validation object of type undefined was created for it in accessibility.json, same applies for other similar cases
```
"12_FOR_BACKGROUNDOPACITY_IN_CLOSEDCAPTIONS_SETTINGS": {
        "method": "accessibility.closedCaptionsSettings",
        "validationJsonPath": "result.styles.backgroundOpacity",
        "expected": "INTEGER12"
    },
```
    whereas for backgroundColor, we do have an existing validation object in accessibility.json pointing to a default value, so for such other cases, we have only updated existing object to type undefined without creating a new object
```
     "#000000_FOR_BACKGROUNDCOLOR_IN_CLOSEDCAPTIONS_SETTINGS_EVENT": {
        "event": "accessibility.onClosedCaptionsSettingsChanged",
        "validationJsonPath": "eventResponse.styles.backgroundColor",
        "expected": "ACCESSIBILITY_BACKGROUNDCOLOR_000000"
    },
```